odl/set/domain.py

```diff
@@ -431,7 +432,7 @@ def measure(self, ndim=None):
         elif ndim > self.true_ndim:
             return 0.0
         else:
-            return np.prod((self.max_pt - self.min_pt)[self.inondeg])
+            return np.prod(self.extent()[self.nondegen_byaxis])
```

Comment:
Probably because the thought was that for some grid or domain structures in the future, this might be costly to compute. If we're sure that it's always gonna be cheap, we can make it a property (but then what to do with `min` and `max` which are methods, too?).